libkrun/libkruntime.c

```diff
@@ -40,4 +41,37 @@ JNIEXPORT jdouble JNICALL
 Java_IterationsRunner_JNI_1clock_1gettime_1monotonic(JNIEnv *e, jclass c) {
     return (jdouble) clock_gettime_monotonic();
 }
+
+JNIEXPORT jlong JNICALL
+Java_IterationsRunner_JNI_1read_1ts_1reg(JNIEnv *e, jclass c)
+{
```

Comment:
Perhaps I'm just confused, but it looked to me like we have a couple of functions with a different layout to the rest. e.g. lines 60--66:

```c
u_int64_t
read_ts_reg() {
    u_int32_t hi, lo;

    __asm volatile("rdtsc" : "=d" (hi), "=a" (lo));
    return ((u_int64_t) hi << 32) | (u_int64_t) lo;
}
```